app/assets/javascripts/models/knockout_extensions.js.coffee

```diff
@@ -207,3 +207,18 @@ do (ko, $=jQuery) ->
     ko.bindingHandlers.text.update element, ->
       truncatedText
     return
+
+  ko.bindingHandlers.descriptionLinks =
+    init: (element, valueAccessor, allBindings, viewModel, bindingContext) ->
+      description = $($.parseHTML(bindingContext.$data.description))
+      for el, i in description
+        if el.nodeName == 'A' && el.host != window.location.host
+          $(el).attr('target', '_blank')
+          $(el).attr('class', 'ext')
```

Comment:
I don't think it is likely that we will run across any links with classes in this case. I think `.addClass` is fine.

Reply:
Ok so @jennywagg can you make one last modification please -- change `$(el).attr('class', 'ext')` to `$(el).addClass('ext')` this will keep any existing class the user has specified remains intact.